src/server_manager/web_app/app.ts

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as sentry from '@sentry/electron';
-import * as events from 'events';
+import * as sentry from '@sentry/browser';
```

Comment:
@fortuna, this change broke Sentry. I tested that changing this line to `@sentry/electron` still sends the report, but `@sentry/browser` silently fails to upload it.